omod/src/main/webapp/pages/metadata/roles/templates/retireRoleDialog.gsp

```diff
@@ -0,0 +1,16 @@
+<div class="dialog-header">
+    <h3>Retire Role</h3>
```

Comment:
I thought we just said that roles and privileges are not retireable so you won't actually need this dialog